contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/LicenseStatus.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <Box
+    :kind="boxKind"
+    :loading="isLoading"
+    data-test="license-status"
+  >
+    <template #title>
+      {{ titleText }}
+    </template>
+    <template #description>
+      {{ descriptionText }}
+    </template>
+  </Box>
+</template>
+
+<script>
+  import { computed } from 'vue';
+  import Box from './Box.vue';
+  import { useLicenseNames } from './composables/useLicenseNames';
+  import { communityChannelsStrings } from 'shared/strings/communityChannelsStrings';
+
+  const {
+    licenseCheckPassed$,
+    allLicensesCompatible$,
+    incompatibleLicensesDetected$,
+    channelCannotBeDistributed$,
+    fixLicensingBeforeSubmission$,
+  } = communityChannelsStrings;
+
+  export default {
+    name: 'LicenseStatus',
+    components: {
+      Box,
+    },
+    props: {
+      invalidLicenses: {
+        type: Array,
+        required: false,
+        default: null,
+      },
+      includedLicenses: {
+        type: Array,
+        required: false,
+        default: null,
+      },
+    },
+    setup(props) {
+      const invalidLicenseIds = computed(() => {
+        if (!props.invalidLicenses || props.invalidLicenses.length === 0) {
+          return [];
+        }
+        return props.invalidLicenses;
+      });
```

Comment:
Mostly nitpick, but still want to flag that this could be simplified to be just:
```Js
      const invalidLicenseIds = computed(() =>  props.invalidLicenses || []);
```

And with this simplification ⬆️, we can see that the only thing these computed properties are doing is to set a default value, then thi means that instead of creating these `invalidLicenseIds` and `includedLicenseIds`, a most straightforward way to handle it is to modify the default value of the `props` definition to be:

```js
      invalidLicenses: {
        type: Array,
        required: false,
        default: () => [],
      },
      includedLicenses: {
        type: Array,
        required: false,
        default:() => [],
      },
```
with the default value set.